### PR TITLE
Add media metadata

### PR DESCRIPTION
**Pull request by cvrac**
This one introduces some extra info regarding files and images on the `/admin/media` page by leveraging ActiveStorage functionality.

I also included a new .gitignore rule in order to ignore `.idea` hidden directory :)

### CODE REVIEW

File: app/views/admin/media/_media_table.html.erb [hunk 1]
```diff
@@ -48,7 +48,14 @@
     <% @media.each do |medium| %>
     <tr>
       <td>
-        <a href="<%= url_for(medium.file) %>" rel="noopener" target="_blank"><%= medium.name %></a>
+        <% if medium.file.image? %>
+          <%= image_tag medium.file.variant(resize_to_limit: [100, 100]) %><br>
```
Comment (cvrac) [inline app/views/admin/media/_media_table.html.erb:52]: Let me know if you want a smaller preview

Comment (cvrac) [reply app/views/admin/media/_media_table.html.erb:52]: We could introduce a helper for this one, a method in `Medium` model or a presenter. As a matter of fact on edge rails there's a better way handling this by defining the possible variants in the attachment definition within the model.

Comment (printezisn) [reply app/views/admin/media/_media_table.html.erb:52]: Let's make this 256x256. What do you think ? It's common variant in the admin panel.
If variant definition is available in Rails 6 too, then let's make it a task ;)
Also, how about making the image a link too ? same as the one for the image name.

File: app/views/admin/media/_media_table.html.erb [hunk 2]
```diff
@@ -48,7 +48,14 @@
...
+        <% end %>
+        <a href="<%= url_for(medium.file) %>" rel="noopener" target="_blank"><%= medium.name %></a><br>
+        <span><%= number_to_human_size(medium.file.byte_size) %></span><br>
+        <% if medium.file.image? %>
```
Comment (printezisn) [inline app/views/admin/media/_media_table.html.erb:56]: I would suggest to make sure that both width and height exist. Sometimes sidekiq takes time to calculate the metadata (and sometimes it is down :P)

File: app/views/admin/media/_media_table.html.erb (hunk 1 above)
Comment (cvrac) [reply app/views/admin/media/_media_table.html.erb:52]: Unfortunately [it isn't,](https://edgeguides.rubyonrails.org/active_storage_overview.html#:~:text=You%20can%20configure%20specific%20variants%20per%20attachment%20by%20calling%20the%20variant%20method%20on%20yielded%20attachable%20object%3A) but we could take a look by monkey-patching it here in the future.

Sure, I will take care of both

**Comment by cvrac** [comment]
2021-11-20-183439_1911x696_scrot

**Review by printezisn — APPROVED** [review]
👍 👍

**Pull request merged**